rclcpp_action/src/server.cpp

```diff
@@ -184,9 +184,9 @@ class ServerBaseImpl
 
               auto ret2 = rcl_timer_get_time_until_next_call(expire_timer, &time_until_call);
               if (ret2 == RCL_RET_TIMER_CANCELED) {
-                rclcpp::Time endless(std::chrono::duration_cast<std::chrono::nanoseconds>(
-                  std::chrono::hours(100)).count(), clock_type);
-                expire_cv->wait_until(l, endless, [this] () -> bool {
+                rclcpp::Duration endless(std::chrono::duration_cast<std::chrono::nanoseconds>(
+                  std::chrono::hours(100)));
```

Comment:
I believe the `duration_cast` isn't necessary here; `std::chrono::hours` should work directly.

Reply:
fixed, should I we restart the CI ?

Reply:
although apparently this PR (as-is) might not fix the issue: https://github.com/ros2/rclcpp/issues/2799#issuecomment-2794482601

Reply:
It does fix it after all! My bad, I was drawing conclusions too quickly.